boards/DCM/Inc/Io/Io_LSM6DS33.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+/**
+ * Driver for interfacing with the IMU LSM6DS33 via I2c
+ *
+ * The Application Note for this IMU can be found here:
+ * https://www.pololu.com/file/0J1088/LSM6DS33-AN4682.pdf
+ */
+
+#include <stdint.h>
+#include <stm32f3xx_hal.h>
+#include <stdbool.h>
+#include <App_SharedExitCode.h>
+
+/**
+ * Update acceleration and angular rate values for the given Imu.
+ */
+void Io_LSM6DS33_ExtiCallback(void);
```

Comment:
Is there any point to keeping the EXTI callback stuff if we're spoofing all the Io code? I've been thinking maybe it's better to remove all the interrupt related stuff for this PR

Reply:
Yeah this is a good idea. We can add this later.

Reply:
Great done